manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -90,6 +92,7 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--cloud-config=$(CLOUD_CONFIG)"
             - "--cluster=$(CLUSTER_NAME)"
+            - "--v=5"
```

Comment:
For production deployments , I don't think enabling verbose by default is recommended , as right now we dont have different release and development manifest

Reply:
ok, I will remove this one